yarn.lock

```diff
@@ -5701,6 +5739,23 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask/multichain@npm:^1.1.0":
+  version: 1.1.0
+  resolution: "@metamask/multichain@npm:1.1.0"
+  dependencies:
+    "@metamask/api-specs": "npm:^0.10.12"
+    "@metamask/controller-utils": "npm:^11.4.3"
+    "@metamask/eth-json-rpc-filters": "npm:^7.0.0"
```

Comment:
This is quite an old copy of this package. The extension is on v9. We're re-introducing `@metamask/json-rpc-engine@8` to our dependency tree due to this.

Not a blocker, but if you could update this in core, it would be appreciated 🙏

Reply:
https://github.com/MetaMask/core/pull/5040

Reply:
This has been released! We can update it here to remove the extra packages

Reply:
done here https://github.com/MetaMask/metamask-extension/pull/27847/commits/0ffeec2d4ce6189bd9b23cc9a3d2b82e90baf66d